app/classifier/tasks/translations.spec.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import assert from 'assert';
+import { mount } from 'enzyme';
+import TaskTranslation from './translations';
+import translations from '../../pages/project/translations';
+
+function StubTask() {
+  return (
+    <p> I do nothing.</p>
+  );
+}
+
+translations.strings.workflow.tasks = {
+  'survey.choices.ar.label': 'Translated Armadillo',
+  'survey.questions.ho.answers.one.label': 'Translated 1'
+};
+
+const task = {
+  choices: {
+    ar: {
+      label: 'Armadillo'
+    },
+    to: {
+      label: 'Tortoise'
+    }
+  },
+  questions: {
+    ho: {
+      answers: {
+        one: {
+          label: '1'
+        },
+        two: {
+          label: '2'
+        }
+      }
+    }
+  }
+};
+
+const expectedTranslation = {
+  choices: {
+    ar: {
+      label: 'Translated Armadillo'
+    },
+    to: {
+      label: 'Tortoise'
+    }
+  },
+  questions: {
+    ho: {
+      answers: {
+        one: {
+          label: 'Translated 1'
+        },
+        two: {
+          label: '2'
+        }
+      }
+    }
+  }
+};
+
+describe('Task translation', function () {
+  const wrapper = mount(
+    <TaskTranslation taskKey='survey' task={task} >
+      <StubTask task={task} />
+    </TaskTranslation>
+  );
+  const stubTask = wrapper.find(StubTask);
+
+  it('should not mutate the original task', function () {
```

Comment:
I should check that this actually fails if `TaskTranslation` does mutate the `task` object.

Reply:
It didn't fail when `task` was mutated, so I've fixed that.